lib/tty/commands/new.rb

```diff
@@ -147,8 +147,11 @@ def execute(input: $stdin, output: $stdout)
         make_executable
         output.puts install_info.join("\n") unless install_info.empty?
 
-        output.print "\n" + @pastel.green("Your teletype project has been created successfully.\n")
-        output.print "\n" + @pastel.green("Run \"teletype help\" for more commands.\n")
+        output.print "\n" + @pastel.green("Your teletype project has been created successfully in directory \"#{app_path}\".\n")
+        output.print "\n" + @pastel.yellow("Before you can begin working, you'll need to modify the \"#{app_path}.gemspec\" file:\n")
+        output.print @pastel.yellow("  - Replace all TODO: text with valid information (summary, description, etc)\n")
+        output.print @pastel.yellow("  - Replace all metadata information (URIs, hosts) with valid URLs, or delete them.\n")
```

Comment:
One addition I would have is to include the third bullet to advise about suggested tty components automatically injected into gemspec to pick & choose. Maybe something like "Uncomment tty components that you wish to use in your app and delete remaining"?

Reply:
Good idea, I'll throw that on there!

Reply:
(also, looking into the failed test now)